radioClass.py

```diff
@@ -23,6 +24,8 @@ class Radio():
     
     # Valid control modes for the radio
     controlModes = ["None",             # No PTT control
+                    "XCMP-XPR",         # eXtended Control and Management Protocol for the XPR series radios
+                    "XCMP-APX",         #TODO: XCMP for APX 
```

Comment:
I'd say don't add the APX option until we have interface classes to support it

Reply:
Fair, changing that